packages/web/src/services/audius-backend/Tipping.ts

```diff
@@ -58,3 +61,71 @@ export const fetchSupporters = async ({
     return []
   }
 }
+
+export type UserTipRequest = {
+  userId?: string
+  limit?: number
+  offset?: number
+  receiverMinFollowers?: number
+  receiverIsVerified?: boolean
+  currentUserFollows?: 'sender' | 'receiver' | 'sender_or_receiver'
+  uniqueBy?: 'sender' | 'receiver'
+  minSlot?: number
+  maxSlot?: number
+  txSignatures?: string[]
+}
+type UserTipOmitIds = 'sender_id' | 'receiver_id' | 'followee_supporter_ids'
+type UserTipResponse = Omit<UserTip, UserTipOmitIds> & {
+  sender: any
+  receiver: any
+  followee_supporters: any[]
+}
+
+export const fetchRecentUserTips = async ({
+  userId,
+  limit,
+  offset,
+  receiverMinFollowers,
+  receiverIsVerified,
+  currentUserFollows,
+  uniqueBy,
+  minSlot,
+  maxSlot,
+  txSignatures
+}: UserTipRequest): Promise<UserTipResponse[]> => {
+  try {
+    await waitForLibsInit()
+    const response = await libs().discoveryProvider._makeRequest({
```

Comment:
Any reason we're calling `_makeRequest` directly? This probably should live in APIClient with the rest of our DN interface. Plus that would mean we don't have to expose all of these query params which we're not currently using to the rest of the client code

Reply:
hmm did we not want to have more separated and domain specific files for api requests?